src/asmcnc/skavaUI/screen_stop_or_resume_decision.py

```diff
@@ -133,17 +133,34 @@ def __init__(self, **kwargs):
     
     def popup_help(self):
 
-        info = (
-            self.l.get_bold('Cancel')  + '[b]' +  " (X)" +'[/b]' + \
-            "\n" + \
-            self.l.get_str("Pressing cancel will cancel the job.") + \
-            "\n\n" + \
-            self.l.get_bold('Resume') + '[b]' +  " (>)" +'[/b]' + \
-            "\n" + \
-            self.l.get_str("Pressing resume will continue the job from the point at which it was paused.")
-        )
-
-        popup_info.PopupInfo(self.sm, self.l, 500, info)
+        if 'yetipilot' not in self.reason_for_pause:
+            info = (
+                self.l.get_bold('Cancel')  + '[b]' +  " (X)" +'[/b]' + \
+                "\n" + \
+                self.l.get_str("Pressing cancel will cancel the job.") + \
+                "\n\n" + \
+                self.l.get_bold('Resume') + '[b]' +  " (>)" +'[/b]' + \
+                "\n" + \
+                self.l.get_str("Pressing resume will continue the job from the point at which it was paused.")
+            )
+
+            popup_info.PopupInfo(self.sm, self.l, 500, info)
+        else:
+            info = (
+                self.l.get_bold('Cancel')  + '[b]' +  " (X)" +'[/b]' + \
+                "\n" + \
+                self.l.get_str("Pressing cancel will cancel the job.") + \
+                "\n\n" + \
+                self.l.get_bold('Resume') + '[b]' +  " (>)" +'[/b]' + \
+                "\n" + \
+                self.l.get_str("Pressing resume will continue the job from the point at which it was paused.") + \
+                "\n\n" + \
+                self.l.get_bold('Scan the QR code to learn more about this error.') + \
+                "\n" + \
+                self.l.get_bold("Or visit <URL>").replace('<URL>', 'www.yetitool.com/support > Knowledge Base')
+            )
+
+            popup_info.PopupQRInfo(self.sm, self.l, 500, info, "./asmcnc/skavaUI/img/qr_yetipilot_info.png")
  
```

Comment:
This can be refactored to have the cancel_resume text block just set up once, and then the rest appended depending on need. Doesn't need to block merge though :)